pkg/registry/rbac/rest/storage_rbac.go

```diff
@@ -238,9 +245,11 @@ func (p *PolicyData) EnsureRBACPolicy() genericapiserver.PostStartHookFunc {
 					failedReconciliation = true
```

Comment:
writing to closure state here introduces a data race (same comment applies to all)